### PR TITLE
Orbit description fix

### PR DESCRIPTION
**Pull request by ZachTRice**
Fixes the issue of orbit tracks displaying the wrong measurement descriptions.

I would recommend merging this after merging the [orbit-track-references](https://github.com/nasa-gibs/worldview-options-eosdis/pull/109) PR  in the options repo since some of the code references new parameters; though everything should gracefully fallback regardless.

self.addDescriptions is a big, not so nice looking function providing a lot of fallbacks at the moment. Eventually it can be removed once we have individual layer descriptions in place. Right now it specifically filters out Orbital Tracks to get the right measurement descriptions associated with the orbit layers, then it is also is providing a check to see if a description field is set in the layer file, if not then associate it with the measurement description.
Connects to #477

### CODE REVIEW

**Comment by Benjaki2** [comment]
glad to hear you are thinking about Destroying 💥  that `self.addDescriptions` function. All those nested loops make me dizzy

File: web/js/util/wv.util.js [hunk 1]
```diff
@@ -807,6 +807,11 @@ wv.util = (function(self) {
     return value;
   };
 
+
+  self.capitalizeFirstLetter = function(string) {
```
Comment (Benjaki2) [inline web/js/util/wv.util.js]: could also use lodash's `_.startCase(_.toLower(str));`
stolen from [stackoverflow](https://stackoverflow.com/a/38084493)

Comment (localjo) [inline web/js/util/wv.util.js]: I don't think we should add this as a utility function. It adds more noise and complexity to this file, and makes the code harder to follow. This is a pretty simple operation. We can just do this in `wv.layers.modal.js`;
```
orbitTitle = _.startCase(layer.daynight)  + "/" + _.startCase(layer.track);
```
That's shorter and more clear anyway.

Comment (localjo) [reply web/js/util/wv.util.js]: Also, what case is the string in to begin with?

Comment (ZachTRice) [reply web/js/util/wv.util.js]: Agreed. This makes a lot of sense. I will remove the function and just use it inline since it is so short. <3 lodash

Comment (ZachTRice) [reply web/js/util/wv.util.js]: It is start case. The Daynight and Track need to start with first letter uppercased. _.startCase should be the correct function to use here. Ben's function above makes sure everything is lowercase first before converting to the start case but I don't think all that is needed.

Comment (localjo) [reply web/js/util/wv.util.js]: It's already start case to begin with? Why do we need to convert it?

Comment (ZachTRice) [reply web/js/util/wv.util.js]: Negative. They are lowercase values to begin with. I meant that they need to be start case.

**Pull request merged**